ccmlib/common.py

```diff
@@ -692,3 +693,21 @@ def assert_jdk_valid_for_cassandra_version(cassandra_version):
     if cassandra_version >= '3.0' and get_jdk_version() < '1.8':
         print_('ERROR: Cassandra 3.0+ requires Java >= 1.8, found Java {}'.format(get_jdk_version()))
         exit(1)
+
+
+def aws_bucket_ls(s3_url):
+    bucket_object = s3_url.replace('https://s3.amazonaws.com/', '').split('/')
+    prefix = '/'.join(bucket_object[1:])
+    s3_conn = client('s3')
```

Comment:
we still need to be able to use it without signing, and we need boto3 added to `setup.py` dependency  list 
https://stackoverflow.com/questions/34865927/can-i-use-boto3-anonymously

Reply:
changed

Reply:
@juliayakovlev we still need boto3 define here: https://github.com/scylladb/scylla-ccm/blob/next/setup.py#L27